client/my-sites/plan-features-2023-grid/plan-comparison-grid.tsx

```diff
@@ -364,10 +361,7 @@ const PlanComparisonGridHeaderCell: React.FunctionComponent<
 				</h4>
 			</PlanSelector>
 			<PlanFeatures2023GridHeaderPrice
-				currencyCode={ currencyCode }
-				discountPrice={ planPropertiesObj.discountPrice }
```

Comment:
Hopefully, gradually move away from prop drilling this way. Prefer hooks, context, state, whatever e.g. prices can be fetched from a hook, and ideally, the entire plan properties structure should be decentralized and put together via a series of hooks/components.